tools/webpack.config.ts

```diff
@@ -131,6 +129,52 @@ export default (env?: string) : webpack.Configuration => {
         // re-evaluated as the size of files grows
         devtool: production ? 'source-map' : 'cheap-module-eval-source-map',
     };
+
+    // Run through and add expose-loader to the config
+    let exposeLoaders = [];
+    for(var exposeArr of exposeOptions) {
+        let path;
+        let name = "";
+        let nameArr;
+        let useArr = [];
+        if(typeof exposeArr === "string") {
+            path = exposeArr;
+            nameArr = exposeArr.split('/');
+            nameArr = nameArr[nameArr.length - 1];
+            nameArr = nameArr.split('.');
+            name = nameArr[nameArr.length - 2];
+            useArr.push({loader: 'expose-loader', options: name});
+        } else {
+            path = exposeArr[0];
+            if(Array.isArray(exposeArr[1])) {
+                for(var loaderOptions of exposeArr[1]) {
+                    useArr.push({loader: 'expose-loader', options: loaderOptions})
+                }
+            }  else {
+                    useArr.push({loader: 'expose-loader', options: exposeArr[1]});
+            }
+        }
+        exposeLoaders.push({
+            test: require.resolve(path),
+            use: useArr
+        });
+    }
+    config.module.rules = config.module.rules.concat(exposeLoaders);
```

Comment:
I think `config.module.rules.push(...exposeLoaders)` is more clear and does the same job.